### ISSUE TITLE
snet set default_ipfs_endpoint requires creating an identity

### ISSUE THREAD

**Issue by vsbogd** [singnet/snet-cli#144]
Log
```
Step 67/76 : RUN snet set default_ipfs_endpoint http://localhost:5002
 ---> Running in 1b466b4fb9e6

Please create your first identity by running 'snet identity create'.

The available identity types are:
    - 'rpc' (yields to a required ethereum json-rpc endpoint for signing using a given wallet
          index)
    - 'mnemonic' (uses a required bip39 mnemonic for HDWallet/account derivation and signing
          using a given wallet index)
    - 'key' (uses a required hex-encoded private key for signing)
    - 'ledger' (yields to a required ledger nano s device for signing using a given wallet
          index)
    - 'trezor' (yields to a required trezor device for signing using a given wallet index)
```

**Comment by astroseger** [comment]
It is almost intended behavior. In order to start using snet-cli user should create first identity. There is only one exception: local network can be created before first identity. 

This issue is not important at all. Surely not important enough to change design in which snet-cli is not functional without snet-cli identity (this design simply config.py a little bit).

So I would propose to close this issue.

**Comment by vsbogd** [comment]
I don't understand why one need identity to set `default_ipfs_endpoint` value.

**Comment by astroseger** [comment]
closed with #158
